tests/plugins/pastefromword/generated/excel.js

```diff
@@ -0,0 +1,55 @@
+/* bender-tags: clipboard,pastefromword */
+/* jshint ignore:start */
+/* bender-ckeditor-plugins: pastefromword,ajax,basicstyles,bidi,font,link,toolbar,colorbutton,image */
+/* bender-ckeditor-plugins: list,liststyle,sourcearea,format,justify,table,tableresize,tabletools,indent,indentblock,div,dialog */
+/* jshint ignore:end */
+/* bender-include: _lib/q.js,_helpers/promisePasteEvent.js,_helpers/assertWordFilter.js,_helpers/createTestCase.js */
+/* bender-include: _helpers/createTestSuite.js,_helpers/pfwTools.js */
+/* global createTestSuite,pfwTools */
+
+( function() {
+	'use strict';
+
+	var config = pfwTools.defaultConfig;
+	config.colorButton_normalizeBackground = true;
+	// Firefox adds `border-color:windowtext currentcolor windowtext windowtext;`
+	// for `td` elements with `colspan` attribute.
+	config.disallowedContent = 'td{vertical-align,border-color}';
```

Comment:
Since it adds only `border-color`, what's the reason for adding vertical align too? It is also valuable for us to assert vertical align if possible.